connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -220,11 +220,22 @@ static ConfigDef getConfigDefFromTransformation(String key, Class<?> transformat
         if (transformationCls == null || !Transformation.class.isAssignableFrom(transformationCls)) {
             throw new ConfigException(key, String.valueOf(transformationCls), "Not a Transformation");
         }
+        Transformation transformation;
         try {
-            return (transformationCls.asSubclass(Transformation.class).newInstance()).config();
-        } catch (Exception e) {
+            transformation = transformationCls.asSubclass(Transformation.class).newInstance();
+        } catch (IllegalAccessException | InstantiationException e) {
```

Comment:
This definitely doesn't cover the full space of errors that are possible here -- `asSubclass` could throw a `ClassCastException`, `newInstance` could also throw `SecurityException`. I think the `catch` was broad because this ensures that except for extreme cases like other `Throwables` or `Errors` we get everything converted to `KafkaExceptions`.

Reply:
@ewencp I think this was the intellij formatter. I don't remember making that change. I'll correct it.